app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsConnector.scala

```diff
@@ -32,14 +32,21 @@ class HealthIndicatorsConnector @Inject()(
   import HttpReads.Implicits._
   private val logger = Logger(getClass)
 
+  implicit val rrR: Reads[RepositoryRating] = RepositoryRating.reads
```

Comment:
Scope - this should not be public.
As a top-level member it deserves a better name (even though you only use it implicitly).